storm-client/test/jvm/org/apache/storm/utils/UtilsTest.java

```diff
@@ -94,13 +94,15 @@ public void parseJvmHeapMemByChildOptsTestM() {
         doParseJvmHeapMemByChildOptsTest("Xmx100M results in 100 MB", "Xmx100m", 100.0);
         doParseJvmHeapMemByChildOptsTest("Xmx100m results in 100 MB", "Xmx100M", 100.0);
         doParseJvmHeapMemByChildOptsTest("-Xmx100M results in 100 MB", "-Xmx100m", 100.0);
+	doParseJvmHeapMemByChildOptsTest("-Xmx2048M results in 2048 MB", "-Xmx2048m", 2048.0);
     }
 
     @Test
     public void parseJvmHeapMemByChildOptsTestG() {
         doParseJvmHeapMemByChildOptsTest("Xmx1g results in 1024 MB", "Xmx1g", 1024.0);
         doParseJvmHeapMemByChildOptsTest("Xmx1G results in 1024 MB", "Xmx1G", 1024.0);
         doParseJvmHeapMemByChildOptsTest("-Xmx1g results in 1024 MB", "-Xmx1g", 1024.0);
+	doParseJvmHeapMemByChildOptsTest("-Xmx2g results in 2048 MB", "-Xmx2g", 2048.0);
```

Comment:
Another nit: Would be nice to fix the UtilsTest#80 so that parameters are in correct order - first expected, then actual. Otherwise error message is confusing.